models/reviews.js

```diff
@@ -3,16 +3,19 @@ const db = require("../db/connection.js");
 
 exports.fetchReview = (review_id) => {
   return db
-    .query(`SELECT * FROM reviews WHERE review_id = $1; `, [review_id])
+    .query(
+      `SELECT reviews.*, COUNT(comments.review_id) ::INT AS comment_count FROM reviews LEFT JOIN comments ON comments.review_id = reviews.review_id WHERE reviews.review_id = $1 GROUP BY reviews.review_id;`,
```

Comment:
beacause you're using backticks `\`\` here  you can put your db query on multiple lines!

I think it's more readable this way

```js
  `SELECT 
      reviews.*, 
      COUNT(comments.review_id) ::INT AS comment_count 
   FROM 
      reviews 
   LEFT JOIN 
      comments ON comments.review_id = reviews.review_id 
   WHERE 
      reviews.review_id = $1 GROUP BY reviews.review_id;`
```

(or something...!)